src/app/views/jobs/jobs.component.ts

```diff
@@ -27,19 +36,41 @@ export class JobsComponent extends TranslationListener implements OnInit {
 
   ngOnInit() {
     this.loadData();
+
+    this.initLocation();
   }
 
   loadData() {
-    this.jobProxy.getJobs({include: 'owner,company,hourly_pay,company.company_images', 'filter[filled]': false, 'page[number]': this.page.toString()})
+    this.jobProxy.getJobs({ include: 'owner,company,hourly_pay,company.company_images', 'filter[filled]': false, 'page[number]': this.page.toString() })
       .then(result => {
         this.jobs = result.data;
         this.totalJobs = result.total;
       });
   }
 
+  initLocation() {
+    this.geolocationService.getLocation().subscribe(
+      position => {
+        this.mapUserLocation = new MapLocation({
+          longitude: position.coords.longitude,
+          latitude: position.coords.latitude
+        });
+        this.mapLocation = this.mapUserLocation;
+        this.mapZoom = 12;
+      },
+      error => {
+        console.log(error);
```

Comment:
Well an error means that the users location can't be established and hence the mapUserLocation will in this case be uninitialized. No user marker will appear in the map. Which in the case of disallowing geolocation is exactly what you would expect but you can still use the map to check for jobs.

So the error is not fatal enough to alert about in my opinion, (in the case of malfunction), the user marker will just be missing. But could drop the log statement and also the error messages since they wont be displayed. Or just change the error messages so they are more log appropriate. What would be preferable?